docs/usage/dependency-dashboard.md

```diff
@@ -0,0 +1,152 @@
+
+## How to enable the dashboard
+
+<!-- TODO: It might be nice to change our config presets, so we have one to `:enableDependencyDashboard` and one to `:disableDependencyDashboard`. -->
+
+To turn on the Dashboard manually, add the `:dependencyDashboard` preset to your `extends` array in the Renovate configuration file:
+
+```json
+{
+  "extends": ["config:base", ":dependencyDashboard"]
+}
+```
+
+Or set `dependencyDashboardApproval` to `true`:
+
+```
+{
+  "dependencyDashboardApproval": true
+}
+```
+
+<!--
+TODO: discuss whether we need to change things.
+
+https://docs.renovatebot.com/configuration-options/#dependencydashboardapproval
+says to set `dependencyDashboardApproval` to `true`, but we also have the `:dependencyDashboard`, which is not mentioned in the link.
+
+-->
+
+## How to disable the dashboard
+
+<!-- TODO: It might be nice to change our config presets, so we have one to `:enableDependencyDashboard` and one to `:disableDependencyDashboard`. -->
+
+To disable the Dependency Dashboard, set `dependencyDashboard` to `false`.
+
+```json
+{
+  "extends": ["config:base"],
+  "dependencyDashboard": false
+}
+```
+
+## Usecases
+
+This section explains some common usecases where having the Dependency Dashboard can help.
+
+### Visibility into rejected/deferred updates
+
+Renovate's Dependency Dashboard shows a overview of all updates that are still "to do".
+
+If you close a update PR from Renovate, the Dashboard will list this update.
+If you later change your mind about the update, you can get a new PR by clicking the corresponding checkbox on the dashboard.
+
+### Show errored out updates
+
+<!-- TODO: Not sure if showing errored updates/branches require the dashboard? Do we open a new issue to warn about a branch error? -->
+
+### Dependency Dashboard Approval workflow
+
+Sometimes you want Renovate to wait for your approval before creating a update PR.
+You can customize this "wait for approval" behavior however you like best.
+
+At a high level the options are:
+
+- Require approval for _all_ updates
+- Require approval for a type of updates (`major` for example)
+- Require approval for specific packages
+
+You can mix and match these options as well.
+
+#### Require approval for all updates
+
+We do not recommend that you require approval for _all_ updates.
+When you require prior approval, you need to check the dashboard issue regularly to check for important updates.
+You'll probably forgot to check often enough, out of sight is out of mind!
+
+Maybe you find Renovate too noisy, and want to opt-out of getting automatic updates whenever they're ready.
+
+In this case, you can tell Renovate to wait for your approval before making any pull requests.
+This means that you have full control over when you get updates.
+
+<!-- TODO: question: do you still get security updates when you tell Renovate to wait for approval for all updates? -->
```

Comment:
Question about security updates when you require approval for _all_ updates.

Reply:
I think approval for all means approval for all, even security updates.

I think we shouldn't show how to enable approval for **all**, as we strictly discourage it.

Reply:
> I think approval for all means approval for all, even security updates.

Hmm, at least the behavior is consistent! 😄